packages/react/src/components/DetailsList/DetailsRow.base.tsx

```diff
@@ -308,6 +313,11 @@ export class DetailsRowBase extends React.Component<IDetailsRowBaseProps, IDetai
         aria-selected={ariaSelected}
         allowFocusRoot={true}
       >
+        {ariaRowDescription ? (
+          <span key="description" id={this._ariaRowDescriptionId} className={this._classNames.screenReaderOnly}>
```

Comment:
```suggestion
          <span key="description" role="presentation" id={this._ariaRowDescriptionId} className={this._classNames.screenReaderOnly}>
```

Reply:
So it doesn't throw off the row/gridcell hierarchy & cell count. Otherwise looks good!